tests/test_spiped.sh

```diff
@@ -106,7 +106,7 @@ setup_spiped_encryption_server () {
 		-s [127.0.0.1]:${src_port}	\
 		-t [127.0.0.1]:${mid_port}	\
 		-p ${s_basename}-spiped-e.pid	\
-		-k /dev/null -o 1
+		-D -k /dev/null -o 1
```

Comment:
This is weird, and does exactly the opposite of what I think you're trying to do.